src/chapters/orm-relationships/many-to-many.rst

```diff
@@ -0,0 +1,332 @@
+- **Line 120**: Passes the DTO into the view.
+- **Line 121**: Returns the name of the template containing the form.
+
+While this may seem like a lot of new concepts, it really isn't. If you look closely, the one new thing that we are doing is using a DTO class to bind an *existing* event to the form. The other steps here are variations of thing you have done before.
+
+Now let's look at the form in ``events/add-tag.html``.
+
+.. sourcecode:: html
+   :linenos:
+
+   <!DOCTYPE html>
+   <html lang="en" xmlns:th="http://www.thymeleaf.org/">
+   <head th:replace="fragments :: head"></head>
+   <body class="container">
+
+   <header th:replace="fragments :: header"></header>
+
+   <form method="post">
+      <div class="form-group">
+         <input type="hidden" th:field="${eventTag.event}">
+         <select th:field="${eventTag.tag}">
+               <option th:each="tag : ${tags}"
+                     th:value="${tag.id}"
+                     th:text="${tag.name}"
+               ></option>
+         </select>
+      </div>
+      <input type="submit" class="btn btn-success" value="Add Tag">
+   </form>
+
+   </body>
+   </html>
+
+This form has two inputs. The first---with ``th:field="${eventTag.event}"``---is hidden, since it should not be modified by the user. We use it to keep track of the specific event that we are about to add a tag to.
+
+The second field, the ``select`` element, is bound to the ``eventTag.tag`` field. The dropdown contains each of the available tags in our application. 
+
+When this form is submitted, it will have all of the information necessary to create an ``EventTagDTO`` object using model binding.
+
+Processing the Form
++++++++++++++++++++
+
+As with ``displayAddTagForm``, we will break down the form's ``POST`` handler in ``EventController`` in detail.
+
+.. sourcecode:: java
+   :lineno-start: 124
+
+   @PostMapping("add-tag")
+   public String processAddTagForm(@ModelAttribute @Valid EventTagDTO eventTag,
+                                 Errors errors,
+                                 Model model){
+
+      if (!errors.hasErrors()) {
+         Event event = eventTag.getEvent();
+         Tag tag = eventTag.getTag();
+         if (!event.getTags().contains(tag)){
+               event.addTag(tag);
+               eventRepository.save(event);
+         }
+         return "redirect:detail?eventId=" + event.getId();
+      }
+
+      return "redirect:add-tag";
+   }
+
+Using model binding, our method takes a valid parameter of type ``EventTagDTO``. Since we referenced the ``event`` and ``tag`` fields of our DTO when rendering the form (in the template, using ``th:field``), our submitted form should contain all of the data necessary to create an ``EventTagDTO`` instance. This instance will be valid if both ``event`` and ``tag`` are non-null. 
```

Comment:
Like we talked about on slack, i'd append this paragraph reinforcing why DTOs are handy - your words:
"The DTO basically allows us to have everything automatically created and validated using model binding. Otherwise, we'd have to manually grab an event ID and a tag ID, query the repositories, validate the results, and so on."

I'd rephrase to something like:

"The reasons for creating a DTO model should hopefully be a bit clearer by now. Using a DTO allows us to create and validate these objects through model binding. The same event and tag relationship information could be processed without a DTO, but this would require passing query parameters for the ids of both event and tag objects, querying the Event Repo and TagRepo for these items, validating those objects, etc. Simply put, the DTO makes this procedure a little cleaner and easier."